src/components/views/emojipicker/EmojiPicker.tsx

```diff
@@ -77,8 +77,8 @@ class EmojiPicker extends React.Component<IProps, IState> {
 
     private scrollRef = React.createRef<AutoHideScrollbar<"div">>();
 
-    private emotes: Map<string, JSX.Element>;
-    private emotesPromise: Promise<Map<string, JSX.Element>>;
+    private emotes!: Map<string, JSX.Element>;
+    private emotesPromise!: Promise<Map<string, JSX.Element>>;
```

Comment:
Same as above

Reply:
switched to optional instead of non null assertion